client/hotrod-client/src/test/java/org/infinispan/client/hotrod/stress/RemoteQueryDslPerfTest.java

```diff
@@ -150,14 +150,12 @@ public void testEmbeddedQueryDslExecution() throws Exception {
    }
 
    public void testEmbeddedLuceneQueryExecution() throws Exception {
-      SearchManager searchManager = org.infinispan.query.Search.getSearchManager(cache);
-      org.apache.lucene.search.Query query = searchManager.buildQueryBuilderForClass(UserHS.class).get()
-            .keyword().onField("name").matching("John1").createQuery();
+      QueryFactory queryFactory = Search.getQueryFactory(cache);
+      Query cacheQuery = queryFactory.create(String.format("From %s where name = 'John1'", UserHS.class.getName()));
 
       final int loops = 100000;
       final long startTs = System.nanoTime();
       for (int i = 0; i < loops; i++) {
-         CacheQuery<User> cacheQuery = searchManager.getQuery(query);
```

Comment:
Extracting this out of the loop is an optimization that impacts the measurement. I believe we want to measure total query execution time including any initial preparation/parsing time for the query. The system is free to eventually internally optimize that by using caching for parsed queries, but we should not help here.

Reply:
ok, I'll move it back